howfairis/Compliance.py

```diff
@@ -3,6 +3,15 @@
 
 # pylint: disable=too-many-arguments
 class Compliance:
+    """Compliance of a repo against the 5 FAIR software recommendations
+
+    Attributes:
+        repository: Whether repository is publicly accessible with version control
+        license: Whether repository has a license
+        registry: Whether code is in a registry
+        citation: Whether software is citable
+        checklist: Whether a software quality checklist is used
```

Comment:
`license` is missing a trailing `_` and `compliant_symbol` and `noncompliant_symbol` are missing in the list of attributes

Reply:
Incorrect as the argument has `license_`, but attribute of the class `self.license` does not.

Added missing attribs.